dist/build-resources.js

```diff
@@ -218,15 +218,21 @@ var moduleRootOverride = {};
 var modulePaths = [];
 var moduleNames = [];
 
+function installedRootModulePaths() {
+  return fileSystem.readdirSync(path.join(optionsGlobal.root, 'node_modules')).map(function (dir) {
```

Comment:
Could you add filtering out hidden directories, like `.bin`, cause they're not actual valid modules?
Thanks! Otherwise looks good.